gotestcover_test.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"github.com/stretchr/testify/assert"
+	"os"
+	"testing"
+)
+
+func TestParseFlags(t *testing.T) {
+	os.Args = []string{"gotestcover",
```

Comment:
There are no overlapping params for go test so this should not have any influence.